src/openms/include/OpenMS/FORMAT/MzTab.h

```diff
@@ -944,7 +944,9 @@ namespace OpenMS
         const std::vector<ProteinIdentification>& prot_ids,
```

Comment:
please add docu that explains the different parameters, especially what the maps do

Reply:
Just FYI, the helper maps are not new. But now just exposed.
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/MzTab.cpp#L1775
I will add further docu.